packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/list/template.html

```diff
@@ -0,0 +1,112 @@
+<oui-datagrid class="table-striped" data-rows="$ctrl.listeners">
+    <oui-datagrid-topbar>
+        <oui-button
+            data-ng-click="$ctrl.goToListenerCreation()"
+            variant="secondary"
+            icon-left="oui-icon-plus"
+        >
+            <span
+                data-translate="octavia_load_balancer_listeners_add_button"
+            ></span>
+        </oui-button>
+    </oui-datagrid-topbar>
+
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_name' | translate"
+        data-property="name"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-prevent-customization
+    >
+        <a data-ng-click="$ctrl.goToListener($row)">
+            {{ ::$value }}
+        </a>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_default_pool' | translate"
+        data-property="defaultPoolId"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-helper=":: 'octavia_load_balancer_listeners_default_pool_helper' | translate"
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_protocol' | translate"
+        data-property="protocol"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="string"
+        data-helper=":: 'octavia_load_balancer_listeners_protocol_helper' | translate"
+    >
+    </oui-datagrid-column>
+    <!--
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_certificate' | translate"
+        data-property="createdAt"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="date"
+    >
+    </oui-datagrid-column>
+    -->
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_port' | translate"
+        data-property="port"
+        data-sortable
+        data-searchable
+        data-filterable
+        data-type="number"
+        data-helper=":: 'octavia_load_balancer_listeners_port_helper' | translate"
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'octavia_load_balancer_listeners_provisioning_status' | translate"
+        data-property="provisioningStatus"
+        data-sortable
+        data-type="options"
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_success': $value === 'active' || $value === 'deleted',
```

Comment:
```suggestion
                'oui-badge_success': ['active', 'deleted'].includes($value),
```

Reply:
Wouldn't it be even better to keep the way I did it for the loadbalancer detail: having constants for the mapping ?
See: https://github.com/ovh/manager/pull/10183/files#diff-5d9ae4ea3f3f8fe8830a0472d92b35001c637c8c30e3e57a44c0203e910b9d25
and https://github.com/ovh/manager/pull/10183/files#diff-30c8f6c4c4f3dfb557f25f5ef6a09fda40e166fe14bb07d46d486510770a7e8e (line 75)

Reply:
Yes I agree